src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -2102,7 +2102,7 @@ public void TestRegionWithSingleLineComment()
                 new DirectiveInfo { Kind = SyntaxKind.EndRegionDirectiveTrivia, Status = NodeStatus.IsActive });
 
             var regionDirective = (RegionDirectiveTriviaSyntax)node.GetFirstDirective();
-            Assert.Equal($"#region A//B{Environment.NewLine}", regionDirective.ToFullString());
```

Comment:
afaict, the prior behavior was just incorrect.  The newline is never part of directive, but because we were scanning tokens *within* the end of this directive, we then started lexing the `//` as a comment trivia =-o, which then ate up through the end of line.